lib/load-changelog-config.js

```diff
@@ -25,19 +25,13 @@ export default async ({ preset, config, parserOpts, writerOpts, presetConfig },
 
   if (preset) {
     const presetPackage = `conventional-changelog-${preset.toLowerCase()}`;
-    loadedConfig = importFrom.silent(__dirname, presetPackage) || importFrom(cwd, presetPackage);
+    loadedConfig = await (importFrom.silent(__dirname, presetPackage) || importFrom(cwd, presetPackage))(presetConfig);
   } else if (config) {
-    loadedConfig = importFrom.silent(__dirname, config) || importFrom(cwd, config);
+    loadedConfig = await (importFrom.silent(__dirname, config) || importFrom(cwd, config))();
   } else {
-    loadedConfig = conventionalChangelogAngular;
+    loadedConfig = await conventionalChangelogAngular();
```

Comment:
the angular config currently does not accept config, so i think it is ok to avoid passing the config here